fixieai/client/client.py

```diff
@@ -166,3 +168,28 @@ def create_session(self) -> Session:
     def get_session(self, session_id: str) -> Session:
         """Return an existing Session object."""
         return Session(self, session_id)
+
+    def get_current_username(self) -> str:
+        """Returns the username of the current user."""
+        query = gql(
+            """
+            query getUsername {
+                user {
+                    username
+                }
+            }
+        """
+        )
+        result = self._gqlclient.execute(query)
+        assert "user" in result and isinstance(result["user"], dict)
+        user = result["user"]
+        assert "username" in user and isinstance(user["username"], str)
+        return user["username"]
+
+    def refresh_agent(self, agent_handle: str):
+        """Indicates that an agent's prompts should be refreshed."""
+        username = self.get_current_username()
+        requests.post(
+            f"{self._api_url}/api/refresh/{username}/{agent_handle}",
+            headers=self._request_headers,
```

Comment:
FYI `/refresh` currently can be hit unauthed. Something we can fix if we put refresh as part of cli, but maybe not if we keep it as part of `.serve()`.

Reply:
This is true -- I'm inclined to leave the auth headers here because it feels sort of weird that it's not authed (i.e. eventually we might be able to make it authed)? But I can remove it if that's just confusing.

Reply:
Agreed. Let's keep it in.